olclient/schemata/shared_definitions.json

```diff
@@ -36,7 +36,17 @@
     "additionalProperties": false,
     "properties": {
       "url":   { "type": "string" },
-      "title": { "type": "string" }
+      "title": { "type": "string" },
+      "type": {
```

Comment:
Is type required?

Reply:
Good catch, but unfortunately this was deliberate -- in an ideal world `type` should be required, but there are many current links that don't have it, so I decided to leave `type` as optional to avoid flagging a whole lot of records as trivially broken for now. I'm happy to revisit that decision if you think we should be stricter. This is another case of  finding that balance between documenting the data as it is vs. as it should be!